pkg/cli/mirror/copy.go

```diff
@@ -193,7 +196,16 @@ func (o *MirrorOptions) bulkImageMirror(ctx context.Context, isc *v1alpha2.Image
 
 		klog.Infof("Processing contents of local catalog %s\n", operator.Catalog)
 
-		relatedImages, err := getRelatedImages(catalogContentsDir, operator.Packages)
+		cfg, err := o.getFilteredDeclarativeConfig(ctx, operator, catalogContentsDir)
+		if err != nil {
+			klog.Errorf("getFilteredDeclarativeConfig error: %s", err.Error())
+			if strings.Contains(err.Error(), "package does not exist in new model") {
```

Comment:
Thanks @jchunkins for your suggestion. This is going to be handle in future refactoring since the legacy code does not have defined error types for all kind of errors right now.